lib/presentation/home/widgets/weather_days_list.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:db_course_app/resources/images.dart';
+import 'package:flutter/material.dart';
+
+const TextStyle textStyle = TextStyle(
+  fontSize: 18,
+  fontWeight: FontWeight.w400,
+);
+
+const TextStyle textStylePrecipitation = TextStyle(
+    color: Color.fromRGBO(1, 14, 130, 0.5),
+    fontSize: 12,
+    fontWeight: FontWeight.w400);
+
+class WeatherListItem extends StatelessWidget {
+  const WeatherListItem({
+    required this.temperature,
+    required this.weekDay,
+    required this.weatherImagePath,
+    this.precipitation,
+    this.last = false,
+    Key? key,
+  }) : super(key: key);
+
+  final String temperature;
+  final String weekDay;
+  final String weatherImagePath;
+  final int? precipitation;
+  final bool last;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      margin: const EdgeInsets.fromLTRB(0, 0, 0, 8),
```

Comment:
If you need only one element for insets use named constructor: 

`const EdgeInsets.only(bottom: 8)`